test/blueprints/Do.Test.Blueprints.Service/Business/Singleton.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Do.Communication;
+using Do.Database;
+using Newtonsoft.Json;
```

Comment:
remove extra usings

Reply:
add following to `.editorconfig` right before `# Stylecop`

```
# CS

dotnet_diagnostic.CS1591.severity = none # Missing XML comment
```

and enable `<GenerateDocumentationFile>true</GenerateDocumentationFile>` in `Directory.Build.props`.

this way `dotnet build` will cause error for unused usings.